### DISCUSSION TITLE
Not all of my expected documents are coming through the PNP Results

### DISCUSSION THREAD

**Discussion by nicolesherman00** [microsoft-search/pnp-modern-search#2210] [Q&A] [answered]
My Query Template in the results webpart is using a path to a document library.
All that is included in my template is:
{searchTerms}
Path:https://chashealth.sharepoint.com/CHAS Job Aid

But not all of my documents are coming through. 
I have installed the latest version of PNP Modern Search.

When I search by Filter I get 14 results.
![image](https://user-images.githubusercontent.com/22782498/177650216-fc6f4523-10e5-475d-afa4-4f1b70307533.png)

But the folder that have the default value set to that filter item is showing 21 items.  
![image](https://user-images.githubusercontent.com/22782498/177650435-bb2175de-a91f-4d46-b0a3-f64b18c4e557.png)

Or, if I search by search bar, it only brings in this document when I search for pest:
![image](https://user-images.githubusercontent.com/22782498/177650591-ef4ee4e1-1856-4408-b6b7-ee93c9ec9ba7.png)

It will not find this item when searching or when filtering into the filterable column:
![image](https://user-images.githubusercontent.com/22782498/177650716-a58afae7-f632-4833-92ac-745ebf828830.png)

Some guidance/help would be so appreciated.

**Comment by wobba** [comment, marked as answer]
Can you share the path of you library?
`Path:https://chashealth.sharepoint.com/CHAS Job Aid` translates to `path:https://chashealth.sharepoint.com/CHAS AND job AND aid`. If you add quotes then it matches the path. `Path:"https://chashealth.sharepoint.com/CHAS Job Aid"` And you have to add the sitename to the path as well. Basically navigate to the Library and copy the URL and past it in.. similar to `path:https://tenant.sharepoint.com/sites/foo/Shared%20Documents"`

Next, the count returned is an estimate when coming from search, even at low numbers (unfortunately), and should be raised as a support case with Microsoft and is not an issue in the web parts.

**Reply by nicolesherman00** [reply]
Adding the %20 in the spaces of the path fixed all my issues. Thank you so much!!

**Reply by FranckyC** [reply]
@nicolesherman00 can you mark this reply as the correct answer? Thanks!

**Reply by nicolesherman00** [reply]
@FranckyC - Sorry I am new to using GitHub... how do I mark it?

**Reply by FranckyC** [reply]
You should have a button on the discussion "Mark as answer"

**Reply by FranckyC** [reply]
I converted the issue to QnA and did it for you. I did think it was in the "General" category so you can't mark it as answer :/
